src/capi_frontend/capi.cpp

```diff
@@ -273,7 +273,8 @@ DLL_PUBLIC OVMS_Status* OVMS_ServerStartFromConfigurationFile(OVMS_Server* serve
     ovms::Server* srv = reinterpret_cast<ovms::Server*>(server);
     ovms::ServerSettingsImpl* serverSettings = reinterpret_cast<ovms::ServerSettingsImpl*>(server_settings);
     ovms::ModelsSettingsImpl* modelsSettings = reinterpret_cast<ovms::ModelsSettingsImpl*>(models_settings);
-    auto res = srv->start(serverSettings, modelsSettings, false);
+    serverSettings->withPython = false;
```

Comment:
With ifdef in server_settings this shouldn't be necessary for .so. It would also not block someone from using C-API to start OVMS with MP &Python support and use MP/Python only through gRPC.